GitTfs/Core/GitRepository.cs

```diff
@@ -697,7 +697,7 @@ public bool Checkout(string commitish)
         {
             try
             {
-                _repository.Checkout(commitish);
+                LibGit2Sharp.Commands.Checkout(_repository, commitish);
```

Comment:
Right you are. The change is [here](https://github.com/libgit2/libgit2sharp/commit/7f4a17f973e11796a644dcdea5a3b618e9d326a4). Appears to be a style choice on the dev to not use extension methods.